src/structure/Header.tsx

```diff
@@ -28,7 +28,7 @@ import { ReactComponent as EPNSLogoLight } from './assets/epnsLight.svg';
 
```

Comment:
Same in this file Also Change the components imported from sharedstyling to SharedStylingV2. There are components being used like : Item, ItemH and Section.